src/router.ts

```diff
@@ -1,11 +1,21 @@
 import Vue from "vue";
 import Router from "vue-router";
+
```

Comment:
I like separating the imports like this, but I don't think it makes sense to just do it ad-hoc unless we can enforce the rules with prettier/eslint/other tools. At the very least, if you want to introduce this, you should update all other imports correspondingly to stay consistent.

Reply:
The closest thing I could find was https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md but it doesn't split them. I can either revert this change or split all imports manually in a separate PR.